GiniSwitchSDK/GiniSwitchSDK/Classes/Extraction.swift

```diff
@@ -8,19 +8,19 @@
 
 public class  Extraction {
 
-    let name:String
-    var value:ExtractionValue = ExtractionValue(value: "" as AnyObject, unit: nil)
-    var alternatives:[ExtractionValue] = []
+    public let name:String
+    public var value:ExtractionValue = ExtractionValue(value: "" as AnyObject, unit: nil)
```

Comment:
If I make it a variable, it starts complaining in the init method, because I call other methods to initialise it. It's an annoying Swift thing.